starknet_devnet/starknet_wrapper.py

```diff
@@ -288,7 +290,7 @@ def _update_block_number(self):
         )
         return block_info
 
-    def __get_transaction_handler(self):
+    def __get_transaction_handler(self, external_tx=None):
```

Comment:
Why would the tx be an optional parameter defaulting to None?

Reply:
I choose optional parameter because `_get_transaction_handler` is called by `mock_message_to_l2` and `postman_flush` without external transaction.
Should I changed `mock_message_to_l2` and `postman_flush` implementations?

Reply:
Ah, that would probably be too much work for the scope of this PR.